source/_components/alarmdecoder.markdown

```diff
@@ -44,16 +44,50 @@ alarmdecoder:
       name: 'Front Door'
       type: 'opening'
 ```
-
-Configuration variables:
-
-- **type** (*Required*): The type of AlarmDecoder device: socket, serial or USB
-- **host** (*Optional*): The IP address of the AlarmDecoder device on your home network, if using socket type. Default: `localhost`
-- **port** (*Optional*): The port of the AlarmDecoder device on your home network, if using socket type. Default: `10000`
-- **path** (*Optional*): The path of the AlarmDecoder device, if using socket type. Default: `/dev/ttyUSB0`
-- **baudrate** (*Optional*): The baud rate of the AlarmDecoder device, if using serial type. Default: `115200`
-- **panel_display** (*Optional*): Create a sensor called sensor.alarm_display to match the Alarm Keypad display. Default: `off`
-- **zones** (*Optional*): AlarmDecoder has no way to tell us which zones are actually in use, so each zone must be configured in Home Assistant. For each zone, at least a name must be given. For more information on the available zone types, take a look at the [Binary Sensor](/components/binary_sensor.alarmdecoder/) docs. *Note: If no zones are specified, Home Assistant will not load any binary_sensor components.*
-- **rfid** (*Optional*): The RF serial-number associated with RF zones. Providing this field allows Home Assistant to associate raw sensor data to a given zone, allowing direct monitoring of the state, battery, and supervision status.
-- **relayaddr** (*Optional*): Address of the relay expander board to associate with the zone. (ex: 12, 13, 14, or 15). Typically used in cases where a panel will not send bypassed zones such as motion during an armed home state, the Vista 20P is an example of this. Alarmdecoder can emulate a zone expander board and the panel can be programmed to push zone events to this virtual expander. This allows the bypassed zone binary sensors to be utilized. One example is using bypassed motion sensors at night for motion-based automated lights while the system is armed with the motion sensor bypassed.
-- **relaychan** (*Optional*): Channel of the relay expander board to associate with the zone. (ex: 1, 2, 3, or 4)
+{% configuration %}
+type:
+  description: The type of AlarmDecoder device, optional values: `socket`, `serial` or `USB`
```

Comment:
There's a `:`. Quote the description strong.

Reply:
:( 
Don't see a problem with that to be honest. 
Is there a common standard to follow?

Reply:
The YAML parser sees a `:` and a space pair and thinks that you're trying to define another key. If you quote the string, it will take it literally.